platform/viewer/src/connectedComponents/ViewerMain.js

```diff
@@ -140,9 +144,7 @@ class ViewerMain extends Component {
 
   render() {
     const { viewportSpecificData } = this.props;
-    const viewportData = viewportSpecificData
-      ? Object.values(viewportSpecificData)
-      : [];
+    const viewportData = values(viewportSpecificData);
```

Comment:
Advantage here is the cache we get from `memoize`? We're using `lodash/values` because `Object.values` doesn't work with this pattern?

Reply:
We can use `Object.values` as well. Will have have to use null guard, though.
i.e. 
```
const values = memoize(Object.values);
......
const viewportData = values(viewportSpecificData || {} );
```